src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/add_product_row.html.twig

```diff
@@ -59,7 +59,17 @@
   <td id="addProductTotalPrice"></td>
   {% if orderForViewing.hasInvoice() %}
     <td class="addProductInvoice pr-2">
-      {{ form_row(addProductRowForm.invoice) }}
+      {{ form_row(addProductRowForm.invoice, {
+        'attr': {
+          'data-modal-translations':        {
+            'modal.title': 'Create a new invoice'|trans({}, 'Admin.Actions'),
+            'modal.apply': 'Create'|trans({}, 'Admin.Actions'),
+            'modal.cancel': 'Cancel'|trans({}, 'Admin.Actions'),
+            'modal.close': 'Close'|trans({}, 'Admin.Actions'),
```

Comment:
Does this modal contain both _Cancel_ and _Close_ buttons?
🤔

Reply:
uh no, my bad. Which one should leave?

Reply:
No worries! When there are action verbs at stake, it makes more sense to use the _Cancel_ button. :-)